spec/authority/configuration_spec.rb

```diff
@@ -15,7 +15,8 @@
 
       it "logs to standard error by default" do
         Authority.instance_variable_set :@configuration, nil
-        null = File.exists?('/dev/null') ? '/dev/null' : 'NUL:' # Allow for Windows
+        method = RUBY_VERSION < '3.2.0' ? :exists? : :exist?
```

Comment:
I forgot to mention this earlier, but we should safely be able to use `exist?` with all versions of ruby. `exists?` was just an alias for it!

https://ruby-doc.org/core-2.1.0/File.html#exist-3F-method

Reply:
this was true until the alias was removed in 3.2.

Reply:
Right, we _need_ to use `exist?` in 3.0+, but we can also use it in earlier versions! So we shouldn't need the conditional, we can just use `exist?`. link above is to the 2.1 docs which show that the `exist?` method was already in existence then (lol), and that's the earliest ruby version that the gem still supports